build/ProjectReunion-BuildFoundation.yml

```diff
@@ -9,7 +9,7 @@ parameters:
     default: False
   - name: "IgnoreFailures"
     type: boolean
-    default: False
+    default: True
```

Comment:
Dupe of #2173 ?

Reply:
Was testing the build validation policies. Not meant to be a real PR